assets/charts/control-plane/calico-host-protection/templates/host-endpoint-controller.yaml

```diff
@@ -33,6 +33,11 @@ spec:
       containers:
       - image: quay.io/kinvolk/calico-hostendpoint-controller:v0.0.4
         name: calico-hostendpoint-controller
+        securityContext:
+          privileged: false
```

Comment:
Are all those statements required? Shouldn't just `runAsUser: 65534` be sufficient to make it not run as root?